pipelines/h6c/idr2/v3/lstbin/lstbin-template.toml

```diff
@@ -54,7 +59,7 @@ fileglob = "{night}/zen.{night}.?????.sum{{ DATA_EXTENSION }}.uvh5"
 [LSTAVG_OPTS]
 # Options for the notebook running -- i.e. the metrics and averaging.
 outdir = "/lustre/aoc/projects/hera/h6c-analysis/IDR{{ IDR }}/lstbin-outputs/{{ CASENAME }}/"
-bl_chunk_size = 5000
+bl_chunk_size = "${LSTBIN_OPTS:bl_chunk_size}"
```

Comment:
Why have this at all? I'd just pass in ${LSTBIN_OPTS:bl_chunk_size} where you'd previously pass in ${LSTAVG_OPTS:bl_chunk_size}

Reply:
The `bl_chunk_size` parameter needs to be used both by `hera_opm` and also the notebook itself as a parameter. These both use different parts of this TOML, so I put it into both.